Bindings/bindings.json

```diff
@@ -2,7 +2,8 @@
     "$schema": "<TBD>",
     "contentVersion": "2016-03-04-alpha",
     "variables": {
-        "storageConnStringLabel": "Which storage account should be used?",
+        "storageConnStringLabel": "Storage account connection",
+        "storageConnHelp": "This is the connection string for your storage account.",
         "appSettingsHelp": "You can use &lt;App Setting Name&gt; to reference an App Setting you've already configured",
```

Comment:
Looks like `appSettingsHelp` is no longer used at all. I would just stay with `appSettingsHelp` and keep using it everywhere instead of introducing new variable. This way the change can really be 'just strings'.